src/test/java/pl/edu/zut/mad/schedule/util/DatesProviderTest.kt

```diff
@@ -0,0 +1,36 @@
+package pl.edu.zut.mad.schedule.util
+
+import org.joda.time.Days
+import org.joda.time.LocalDate
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class DatesProviderTest {
+
+    private val startDate = LocalDate.parse("2016-01-01")
+    private val endDate = LocalDate.parse("2017-01-01")
+    private val datesProvider = DatesProvider()
+
+    @Test
+    fun datesAmountIsProper() {
```

Comment:
You could use some more descriptive names with fun \`Here test description\`(). Moreover you could pick some [test naming convention](https://dzone.com/articles/7-popular-unit-test-naming) and use it everywhere